### ISSUE TITLE
Shmanic Config

### ISSUE THREAD

**Issue by lemollon** [ShMaunder/JMapMyLDAP#58]
When  I click on the Shmanic Config under components in Joomla I get a 500 error. The plugin is enabled. when I tested it in the debugger this is what I receive.

"Failed with: <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1"/>
 <title>500 - Internal server error.</title>
 <style type="text/css">
 
 </style>
 </head>
 <body>
 <div id="header"><h1>Server Error</h1></div>
 <div id="content">
 <div class="content-container"><fieldset>
 <h2>500 - Internal server error.</h2>
 <h3>There is a problem with the resource you are looking for, and it cannot be displayed.</h3>
 </fieldset></div>
 </div>
 </body>
 </html>"
  I've looked everywhere for trouble shooting and havent found out what it is wrong. Assistance would really be appreaciated. Thanks!

**Comment by conconnl** [comment]
Enable the PHP errors in your installation and see if you get the real error.

**Comment by lemollon** [comment]
Thank you, I found the error now my LDAP is will no syn with my acitve Direcory. I'm using JMapMyLDAP and the this is there I get from the error log. "[S]12231 - General Error ID for raising a exception log in onLdapSync" I running through all the congifs and I'm still lost.

**Comment by lemollon** [comment]
![image](https://cloud.githubusercontent.com/assets/24685088/22164447/d9d7029a-df1c-11e6-96f9-e159421e4e5a.png)

**Comment by lemollon** [comment]
My name show up through the after loggin in with my AD username and password but when I click on the user in Joomla I get the above error. What am I missing?

**Comment by ShMaunder** [comment]
I assume a later build of Joomla has broken something here which doesn't surprise me too much as its been a while.

I lost all my build VM which used to build & test this extension so I'll take a look this weekend and see whats going on. I'll see if I can get something like Travis to build & test instead.

**Comment by ShMaunder** [comment]
Just in case I cannot replicate this specific issue, when you get the 500 error, does your web server's error log show anything useful?

**Comment by lemollon** [comment]
Yes, I checked the logs and all it says is that username and password do
not match but 100% sure I'm using the right password. Even when I ask to
reset password it send me the reset key. So everything works except for
actually viewing the site. I'm not at work and the exact log is on my work
computer. Is that enough information or would you like screen shots?

**Comment by conconnl** [comment]
@ShMaunder I use the latest Joomla! version and have no problems with the extension itself.
Except it opens to many connections for the cron.

@lemollon 
Before I got every working a year ago, I needed to change some Windows Active Directory configurations. Like enabling LDAPS for password resets, giving the bind user specific rights.
Looking at your XML error it looks like you have tried to configure the Profile plugin and used the Settings Profile Name and Profile Base.
You only need to fill these settings when you want to extend the profile with more AD attributes, but first try without
